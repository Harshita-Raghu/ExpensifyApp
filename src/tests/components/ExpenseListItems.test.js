import React from 'react'
import {shallow} from 'enzyme'
import ExpenseListItems from '../../components/ExpenseListItems'
import expenses from '../fixtures/expenses'

test('should render expense list ',()=>{

    expenses.map((expense)=>{
        const wrapper=shallow(<ExpenseListItems expense={expense}/>)
    expect(wrapper).toMatchSnapshot();
    })
    
})
