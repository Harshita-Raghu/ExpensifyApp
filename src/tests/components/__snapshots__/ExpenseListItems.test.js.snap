// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render expense list  1`] = `
<div>
  <Link
    replace={false}
    to="/edit/undefined"
  />
  <p>
      
  </p>
</div>
`;

exports[`should render expense list  2`] = `
<div>
  <Link
    replace={false}
    to="/edit/undefined"
  />
  <p>
      
  </p>
</div>
`;

exports[`should render expense list  3`] = `
<div>
  <Link
    replace={false}
    to="/edit/undefined"
  />
  <p>
      
  </p>
</div>
`;
